### DISCUSSION TITLE
Do Hooks disable other actions?

### DISCUSSION THREAD

**Discussion by SHxKM** [EPMatt/awesome-ha-blueprints#156] [Q&A]
I’ve successfully set-up a Controller + the Light Hook. Works very nicely. 

I tried mapping the double press event to reset the color of the controlled-light but it seems like Hooks disable this feature? As far as I know the double press action is not mapped to any functionality in the Light hook.

**Comment by EPMatt** [comment]
Hi @SHxKM,

thank you for reporting. Hooks do not disable custom actions supplied in the controller blueprint.
Did you enable virtual double press events for your controller?
They are disabled by default, as reported in the controller docs, since they introduce a slight delay for single short button presses. I'd also like to point out that you can enable virtual double press events just for a single button, leaving the feature disabled for buttons you don't plan to use with double press events. You can read more in your controller's documentation.

Please let me know if enabling virtual double press events solves your issue. :) 
Thanks!
